conf/default-vars.yaml

```diff
@@ -0,0 +1,35 @@
+vars:
+  #families
+  font-primary: 'sans-serif'
+  font-secondary: 'serif'
+  font-heading: 'serif'
```

Comment:
RE mono.  I think that could be added in a separate PR.

In terms of font family. I think its assumed that the parent site would always override these.  I kinda like the idea of it defaulting to the system font declarations though. maybe that could be a separate PR